hack/graph-util.py

```diff
@@ -472,18 +471,17 @@ def get_token(args):
     return None
 
 def set_log_level(args):
-    if args.verbose == "info":
-        _LOGGER.setLevel(logging.INFO)
-        return
-    _LOGGER.setLevel(logging.DEBUG)
+    _LOGGER.setLevel(logging.INFO)
+    if args.verbose == "debug":
+        _LOGGER.setLevel(logging.DEBUG)
```

Comment:
nit: this could be:

```python
_LOGGER.setLevel(args.verbose.upper())
```

without needing a switch on values.  `setLevel` support for string arguments is [new in 3.2][1].  And it works in my Python 2.7.17, although I don't see it [documented for Python 2][2].

[1]: https://docs.python.org/3/library/logging.html#logging.Logger.setLevel
[2]: https://docs.python.org/2/library/logging.html#logging.Logger.setLevel